static/js/src/scores.js

```diff
@@ -16,18 +16,21 @@ function scores() {
                 postScore = Number(postScore);
 
                 var learningObjectiveScore = postScore - preScore;
-                var tableRow = $('<tr></tr>');
+                var tableRow = $('<tr id="' + prefix + '"></tr>');
```

Comment:
HTML id's should be unique.. and seems here a lot of this tr and td code would make duplicate IDs, but maybe I'm wrong? Anyways, the id attr should be change to class, or data attributes, if there are duplicate ids.

Reply:
These should always be unique. The prefix is built up from the index of the map on line 8.

Reply:
Oh I see, cool.

Reply:
I'm using the `[...Array(3).keys]` as a range function, but its confusing, I could make that more clear.